src/Subscriptions/Endpoints/ListSubscriptions.php

```diff
@@ -111,6 +112,11 @@ public function registerRoute()
                             'columns',
                         ],
                     ],
+                    'status' => [
+                        'enum' => [...array_values(SubscriptionStatus::toArray())],
```

Comment:
Let's add support to "any" status this way:

`'enum' => ['all', ...array_values(SubscriptionStatus::toArray())],`